editor/src/core/model/element-metadata-utils.ts

```diff
@@ -680,23 +692,25 @@ export const MetadataUtils = {
   },
   getImmediateChildrenPathsUnordered(
     elements: ElementInstanceMetadataMap,
+    elementPathTree: ElementPathTrees,
     target: ElementPath,
   ): Array<ElementPath> {
     const element = MetadataUtils.findElementByElementPath(elements, target)
     if (element == null) {
       return []
     } else {
       const rootPaths = MetadataUtils.getRootViewPathsUnordered(elements, target)
```

Comment:
Looks like this should be replaced with the Ordered version

Reply:
Fixed in d1565dc9d.